CaughtUp/app/src/main/AndroidManifest.xml

```diff
@@ -18,6 +18,10 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="a0y2v6pCXfyvHgf6HUU8KPdaJ" />
     </application>
 
+    <uses-permission android:name="android.permission.INTERNET" />
```

Comment:
This was part of the Twitter docs to add so I would leave it in there just in case.